cms/server/admin/handlers/task.py

```diff
@@ -502,3 +508,43 @@ def delete(self, task_id):
 
         # Maybe they'll want to do this again (for another task)
         self.write("../../tasks")
+
+
+class DownloadSubmissionsHandler(FileHandler):
+    """Download submissions for a dataset in a zip file.
+    """
+    @require_permission(BaseHandler.AUTHENTICATED)
+    def get(self, task_id):
+        task = self.safe_get_item(Task, task_id)
+
+        zip_filename = "%s.zip" % task.name
+        source_template = "%s_%s%s"
```

Comment:
There's no need to. `submission.language` is the file extension of the language, so it gets added anyway.